### DISCUSSION TITLE
Failed to load resource when route has prefix (Help)

### DISCUSSION THREAD

**Discussion by enkhtulga-gantsagaan** [orchestral/testbench-dusk#80] [Q&A]
- Testbench Version: 7.20.0
- Laravel Version: 9.19
- Version: 8.1.8
- Database Driver & Version:

### Description:
Is it possible to set asset url `http://localhost:8000/front`?. I have routes with `/front` prefix. When I try to run dusk test then assets (images in public directory of Laravel application) are not found 404.
here are some console logs related to asset 404:

```
 {
        "level": "SEVERE",
        "message": "http:\/\/localhost:8080\/front\/images\/not_member\/1.png - Failed to load resource: the server responded with a status of 404 (Not Found)",
        "source": "network",
        "timestamp": 1676010115309
    },
    {
        "level": "SEVERE",
        "message": "http:\/\/localhost:8080\/front\/images\/not_member\/sugo_lp.jpg - Failed to load resource: the server responded with a status of 404 (Not Found)",
        "source": "network",
        "timestamp": 1676010115312
    },
    {
        "level": "SEVERE",
        "message": "http:\/\/localhost:8080\/front\/images\/not_member\/banner.jpg - Failed to load resource: the server responded with a status of 404 (Not Found)",
        "source": "network",
        "timestamp": 1676010115315
    }
```

### Steps To Reproduce:

```
Route::prefix('front')->group(function () {
    Route::get('/users', function () {
        // Matches The "/front/users" URL
    });
    ...
});
```

To run code:
```
php vendor/bin/testbench-dusk package:dusk
```
Thank you for your valuable feedback.

**Comment by enkhtulga-gantsagaan** [comment]
dear contributors, pls help me with above issue. @crynobone @Keoghan

**Comment by Keoghan** [comment]
@enkhtulga-gantsagaan Why is your package trying to load image resources here: `/front/images/...`?
Anything in `public/images` will simply be `/images/...` as per a normal Laravel app.

**Reply by enkhtulga-gantsagaan** [reply]
@Keoghan thank you for your response.

I have a route prefix named `front` as some requirement. Before moving package development. It works on main Laravel application. (Both `/` and `/front`) So I need to migrate this logic to package development without any breaks. 

Thank you for your valuable feedback.

**Reply by enkhtulga-gantsagaan** [reply]
It also works in localhost as normal. Anything in `public/images` will not work in `testbench-dusk`. :(

**Reply by Keoghan** [reply]
I think you need to change your approach for using assets with your package - so you would publish them to the asset directory.

Most packages would publish to public/vendor/[namespace]/images, etc.
